pkg/apis/serving/v1/route_lifecycle_test.go

```diff
@@ -451,8 +451,8 @@ func TestCertificateNotReadyWithBubbledUpMessage(t *testing.T) {
 				Conditions: duckv1.Conditions{
 					{
 						Type:   "Ready",
-						Status: "False",
-						Reason: "CommonName Too Long",
+						Status: "Unknown",
```

Comment:
Why unknown instead of false?

Reply:
Just want to set up different tests for different states.

Reply:
@ReToCode sure, will do this change. Thanks for your idea.

Reply:
Awesome, way better now.